hdf5-vol-daos.spec

```
@@ -31,8 +31,8 @@
 %endif
 
 Name:    hdf5-vol-daos
-Version: 0.1
-Release: 5%{?relval}%{?dist}
+Version: 1.1
+Release: 0rc1%{?relval}%{?dist}
```

Like the comments in the hdf5 PR, the `rc1` is part of the `Version:` not the `Release:`:

```suggestion
Version: 1.1.0~rc1
Release: 1%{?relval}%{?dist}
```

That said, I also like what you did with `%{commit}` in the hdf5 PR.  I'm actually really quite on the fence about that technique vs. externally defining `%{relval}`.  They each have their merits.  But we should pick one and be consistent about it.